cluster/defaults.go

```diff
@@ -71,6 +71,17 @@ const (
 
 	DefaultCanalFlexVolPluginDirectory = "/usr/libexec/kubernetes/kubelet-plugins/volume/exec/nodeagent~uds"
 
+	DefaultApicRefreshTime                           = 1200
+	DefaultOVSMemoryLimit                            = "1Gi"
+	DefaultImagePullPolicy                           = "Always"
+	DefaultServiceMonitorInterval                    = 0
+	DefaultPBRTrackingNonSnat                        = "false"
+	DefaultInstallIstio                              = "false"
+	DefaultIstioProfile                              = "demo"
```

Comment:
Should we set this to `default` and let the user choose if he wants to use `demo` profile?

Reply:
IstioProfile of "demo" is the ACI CNI default. But these settings are for ACI-CNI installed istio, which is disabled by default, and Rancher installed Istio would be preferred for customer usage.